README.md

```diff
@@ -3,3 +3,9 @@ The idea is to learn Django, Django-rest-framework and other relevant technologi
 It consists of the official Django polls application as a base, extended with a restful api. Other features will be added as the training advances.
 
 This proyect uses pipenv for package management.
+
+Features to add in the future:
+    Faker for better test data
+    Patch endpoints for CRUDs
+    Permission per Model restrictions
+    Endpoints for Admins only
```

Comment:
Myabe converting this to a list makes it look better (prepending -)

Reply:
True that, I'll change it on the next PR.